crates/cargo-util-schemas/src/core/partial_version.rs

```diff
@@ -179,9 +183,65 @@ enum ErrorKind {
     Unexpected,
 }
 
-fn is_req(value: &str) -> bool {
-    let Some(first) = value.chars().next() else {
-        return false;
-    };
-    "<>=^~".contains(first) || value.contains('*') || value.contains(',')
+#[cfg(test)]
+mod test {
+    use super::*;
+    use snapbox::str;
```

Comment:
Just a question: we use `snapbox` here just for better assertion message, right?

Reply:
Inline snapshotting!